### ISSUE TITLE
[CATA][Quests][Uldum] Fire From the Sky

### ISSUE THREAD

**Issue by IREvil** [NRG-Wow/NRGWow-Bugtracker#75]
[//]: #  (Only reports in english will be allowed. and specific issues. everything like "ZOMG NOTHING WORKS" will be ignored / deleted)

[//]: # (Please write the titles of the issues following the format:)
[//]: # ([CATA][Quests][Zones] Quest name)
[//]: # ([WOTLK][Spells][Classes] Spell name)
[//]: # ([WOTLK][Talents][Classes] Talent name)
[//]: # ([CATA][Glyphs][Classes] Glyph name)
[//]: # ([CATA][NPCs] NPC name)

**Reporter(Player/Character) Name**: Evilfrostguy

**Description**:  Quest can't be completed because of the vehicle
**Current behaviour**: The vehicle can't be mounted, and killing the mobs individually doesn't count towards the quest progress either

**Expected behaviour**: Vehicle should work properly and I believe something might be wrong with the mobs required as they don't show up as the quest objective

**Steps to reproduce the problem**:

1. Accept quest
2. ???
3. Eh

**Comment by Cannonbow** [comment]
Yes I totally agree with evil  I tried to do this daily quest 2 days ago it happened same think for me normally how it's should work when you accept the quest you need to be summoned to the vehicule to kill those 100 mobs.  But here when u accept the quest nothing is happening and as evil said we can't kill all 100 mobs alone lol that's hard hope you will fix it soon
